.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+For generating a issue, use the below template to make the issue more clear to those contributing to the project
+
+### Expected behavior and actual behavior.
+
+### Steps to reproduce the problem.
+
+### Specifications like the version of the project, operating system, or hardware.
```

Comment:
remove "operating system, or hardware" not sure if it adds value at this point. but good idea!